experiments/dataModels/subscribe_ql.py

```diff
@@ -15,8 +15,15 @@
 
 
 def create_subscription(entity_type, notify_url):
+    if entity_type.startswith('Traffic'):
+        description = "traffic_flow_observed"
+    elif entity_type.startswith('Air'):
+        description = "air_quality_observed"
+    else:
+        description = "description"
+
     s = {
-        "description": "traffic_flow_observed",
+        "description": description,
```

Comment:
How about using the entity_type directly as the description (instead of the ifs)? I'd rather keep it simple, after all it's a simple experiment :)